cli/tests/e2e/snapshots/test_check/test_multiple_configs_file/results.json

```diff
@@ -110,6 +70,46 @@
         "line": 3,
         "offset": 61
       }
+    },
+    {
+      "check_id": "rules.eqeq-is-bad",
+      "end": {
+        "col": 26,
+        "line": 3,
+        "offset": 69
+      },
+      "extra": {
+        "engine_kind": "OSS",
+        "fingerprint": "0x42",
+        "is_ignored": false,
+        "lines": "    return a + b == a + b",
+        "message": "useless comparison operation `a + b == a + b` or `a + b != a + b`; possible bug?",
```

Comment:
weird that there's still some json with 'possible bug?' in it.

Reply:
plus it's weird that this json got reordered in pysemgrep, and osemgrep fails with the new sorting :/

Reply:
there are still some rules (eqeq-meta / eqeq-python / eqeq-source / multiline / bad1 / ...) that include the `possible bug` string, so that output is understandable. but the reordering I don't understand (yet).